client/converter.cpp

```diff
@@ -30,6 +32,58 @@ bool Converter::set_tracepoint_map(const std::string& map_files)
     }
 }
 
+void Converter::_try_setting_mapping_klass_id(const parser::Event& event)
+{
+    if (event.get_klass()->get_id() == HawkTracer::parser::to_underlying(HawkTracer::parser::WellKnownKlasses::EventKlassInfoEventKlass))
+    {
+        if (event.get_value<char*>("event_klass_name") == _mapping_klass_name)
+        {
+            _mapping_klass_id = event.get_value<HT_EventKlassId>("info_klass_id");
+        }
+    }
+}
+
+std::string Converter::_convert_value_to_string(const parser::Event::Value& value)
+{
+    std::string label;
+    if (value.field->get_type_id() == parser::FieldTypeId::UINT64)
```

Comment:
It's just my personal opinion, and feel free to ignore the comment; IMO `switch()` would be more readable here